docs/dev.md

```diff
@@ -62,3 +64,38 @@ Integration tests can read the following system properties to obtain information
+
+Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option. 
+
+The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described below.
+
+You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Maven goal. You can examine the commands used to build and run the container by viewing the console output of dev mode.
+
+When dev mode runs with container support it still provides the same features. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. target/liberty/wlp/usr/servers/defaultServer/logs.
+
+This mode publishes the container ports 9080, 9443 and 7777 by default. If your application needs to publish other ports add them to the `dockerRunOpts` option either in the `pom.xml` file or on the `mvn` command line. E.g. `-DdockerRunOpts="-p 9081:9081"`
+
+Sample configuration of this plugin using the `devc` parameters.
+```
+            <!-- Enable liberty-maven plugin -->
+            <plugin>
+                <groupId>io.openliberty.tools</groupId>
+                <artifactId>liberty-maven-plugin</artifactId>
+                <version>3.3-M1</version>
+                <configuration>
+                    <container>true</container>
+                    <dockerRunOpts>-p 9081:9081</dockerRunOpts>
+                    <dockerfile>myDockerfile</dockerfile>
+                </configuration>
+            </plugin>
+```
+
+| Parameter | Description | Required |
+| --------  | ----------- | -------  |
+| container | If set, run the server in the container described by your Dockerfile. This option works the same as the `devc` goal. The default value is unset or `false` when the Maven goal `dev` is used. | No |
+| dockerRunOpts | This option allows you to add options to the docker run command used by dev mode to launch your server container. For example `-e key=value` is recognized by `docker run` to define an environment variable called `key`. If this option is used the container option is set. | No |
+| dockerfile | Define the file used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default value is `Dockerfile`. If this option is used the container option is set. | No |
```

Comment:
Location of a dockerfile to be used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default value is `Dockerfile`. Setting this parameter defaults the `container` parameter to `true`.

Reply:
Done but worded as "overrides" rather than "defaults" the `container` parameter to `true`